cohere/client.py

```diff
@@ -1051,6 +1052,18 @@ def get_custom_model_by_name(self, name: str) -> CustomModel:
         response = self._request(f"{cohere.CUSTOM_MODEL_URL}/GetFinetuneByName", method="POST", json=json)
         return CustomModel.from_dict(response["finetune"])
 
+    def get_model_metrics(self, custom_model_id: str) -> List[ModelMetric]:
```

Comment:
should we call it get_custom_model_metrics? All of the other functions start with custom_model_....